### ISSUE TITLE
Keyboard scrolling not work

### ISSUE THREAD

**Issue by tiensonqin** [tiensonqin/lambdahackers#70]
Yep, I’m seeing that blue bell right now, yay! :) Thanks for fixing this.

Hey, while we’re at it, can I report another minor thing here? I use SakaKey for keyboard navigation, and keyboard scrolling stopped working after the last update; but the thing is, it doesn’t work ONLY when the page is scrolled up to the top (when the article’s title changes to "Lambdahackers"), in every other situation it works just fine. Also, it doesn’t work ONLY with my custom bindings (i\k), and works just fine with arrows or even Shift+I\Shift+K. I guess the page intercepts that particular bindings, or something like that, is that right?
